sql/src/main/java/io/crate/execution/ddl/SnapshotRestoreDDLDispatcher.java

```diff
@@ -65,7 +55,6 @@
 @Singleton
 public class SnapshotRestoreDDLDispatcher {
```

Comment:
Will drop this class with the next PR (unbound analysis for the restore snapshot statement)